src/main/java/teammates/client/scripts/DataMigrationForIsArchivedAttribute.java

```diff
@@ -54,17 +57,30 @@ protected void doOperation() {
      * @throws InvalidParametersException
      */
     private void setInstructorsIsArchivedInCourse(CourseAttributes course) throws InvalidParametersException, EntityDoesNotExistException {
-        System.out.println("Updating instructors of course: " + course.id);
+        System.out.println("Updating instructors of old archived course: " + course.id);
         
         List<InstructorAttributes> instructorList = logic.getInstructorsForCourse(course.id);
         for (InstructorAttributes instructor: instructorList) {
-            instructor.isArchived = true;
-            logic.updateInstructorByEmail(instructor.email, instructor);    
-            
-            System.out.println("Successfully updated instructor: [" + instructor.email + "] " + instructor.name);
+            if (isPreview) {
```

Comment:
The code looks okay but how about we move this branch to another function having the same level of abstraction? It is for previewing so it should not in the setInstructorsIsArchivedInCourse() function.